src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.Unix.cs

```diff
@@ -30,7 +30,7 @@ public void ResetIdealProcessor()
         private bool PriorityBoostEnabledCore
         {
             get { return false; }
-            set { throw new PlatformNotSupportedException(); }
```

Comment:
Fortunately or unfortunately, depending on how you look at it, no ;)  The Process library is an area where we have a fairly significant test gap.